src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1203,11 +1203,13 @@ private ITupleExpression CreateBoundTupleExpressionOperation(BoundTupleExpressio
 
         private IInterpolatedStringExpression CreateBoundInterpolatedStringExpressionOperation(BoundInterpolatedString boundInterpolatedString)
         {
-            Lazy<ImmutableArray<IInterpolatedStringContent>> parts = new Lazy<ImmutableArray<IInterpolatedStringContent>>(() =>
-                boundInterpolatedString.Parts.SelectAsArray(interpolatedStringContent => CreateBoundInterpolatedStringContentOperation(interpolatedStringContent)));
             SyntaxNode syntax = boundInterpolatedString.Syntax;
             ITypeSymbol type = boundInterpolatedString.Type;
             Optional<object> constantValue = ConvertToOptional(boundInterpolatedString.ConstantValue);
+            Lazy<ImmutableArray<IInterpolatedStringContent>> parts = new Lazy<ImmutableArray<IInterpolatedStringContent>>(() =>
+                boundInterpolatedString.Parts.IsDefaultOrEmpty ?
+                ImmutableArray.Create<IInterpolatedStringContent>(new InterpolatedStringText(new LiteralExpression(string.Empty, syntax, type, constantValue), syntax, type, constantValue)) :
+                boundInterpolatedString.Parts.SelectAsArray(interpolatedStringContent => CreateBoundInterpolatedStringContentOperation(interpolatedStringContent)));
```

Comment:
I agree with Chuck here, it would be suprising if the syntax tree had no parts and the operation tree had one. @dotnet/analyzer-ioperation are we okay with this? If so, I will abandon the PR and close the issue as Won't Fix.